exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -109,19 +121,16 @@ func (e *exporter) serializeMetrics(md pdata.Metrics) []string {
 			for k := 0; k < metrics.Len(); k++ {
 				metric := metrics.At(k)
 
-				var l []string
-				switch metric.DataType() {
-				case pdata.MetricDataTypeNone:
-					continue
-				case pdata.MetricDataTypeGauge:
-					l = serialization.SerializeNumberDataPoints(e.cfg.Prefix, metric.Name(), metric.Gauge().DataPoints(), e.defaultDimensions)
-				case pdata.MetricDataTypeSum:
-					l = serialization.SerializeNumberDataPoints(e.cfg.Prefix, metric.Name(), metric.Sum().DataPoints(), e.defaultDimensions)
-				case pdata.MetricDataTypeHistogram:
-					l = serialization.SerializeHistogramMetrics(e.cfg.Prefix, metric.Name(), metric.Histogram().DataPoints(), e.defaultDimensions)
+				metricLines, err := serialization.SerializeMetric(e.cfg.Prefix, metric, e.defaultDimensions, e.staticDimensions, e.prevPts)
+
+				if err != nil {
+					e.logger.Sugar().Errorf("failed to serialize %s %s, %s", metric.DataType().String(), metric.Name(), err.Error())
+				}
+
+				if len(metricLines) > 0 {
+					lines = append(lines, metricLines...)
+					e.logger.Debug(fmt.Sprintf("Serialized %s %s - %d lines", metric.DataType().String(), metric.Name(), len(metricLines)))
```

Comment:
```suggestion
					e.logger.Debug(fmt.Sprintf("Serialized %s '%s' - %d lines", metric.DataType().String(), metric.Name(), len(metricLines)))
```

Reply:
Do the quotes add anything here? Name will contain no spaces and neither with the type so it is unambiguous

Reply:
Ah I wasn't sure about the type at this point but yeah then it's fine as-is, just wanted to make it consistent with the one above.